card/mod/01_core/set/all/subcards.rb

```diff
@@ -30,28 +33,32 @@ def clear_subcards
   subcards.clear
 end
 
+def deep_clear_subcards
+  subcards.deep_clear
+end
+
 def unfilled?
   (content.empty? || content.strip.empty?) &&
     (comment.blank? || comment.strip.blank?) &&
     !subcards.present?
 end
 
-event :approve_subcards, after: :approve, on: :save do
+def with_id_when_exists card, &block
+  card.director.call_after_store &block
```

Comment:
Ambiguous block operator. Parenthesize the method arguments if it's surely a block operator, or add a whitespace to the right of the `&` if it should be a binary AND.